client-lib-tests/src/fat/java/com/ibm/ws/repository/test/RepositoryUtilsTest.java

```diff
@@ -77,13 +77,13 @@ public void testRepositoryStatusOK() throws IOException, RequestFailureException
      */
     @Test
     public void testInvalidRepositoryNotAvailable() {
-        RestRepositoryConnection invalidLoginInfo = new RestRepositoryConnection("I", "don't", "exist", "http://dont.exist.com");
+        RestRepositoryConnection invalidLoginInfo = new RestRepositoryConnection("I", "don't", "exist", "http://dont252qmadjf842sgs7842d.exist.com");
         assertFalse("An invalid test repository should not be available", invalidLoginInfo.isRepositoryAvailable());
     }
 
     @Test
     public void testInvalidRepositoryThrowsException() throws IOException, RequestFailureException {
-        RestRepositoryConnection invalidLoginInfo = new RestRepositoryConnection("I", "don't", "exist", "http://dont.exist.com");
+        RestRepositoryConnection invalidLoginInfo = new RestRepositoryConnection("I", "don't", "exist", "http://dont252qmadjf842sgs7842d.exist.com");
```

Comment:
This will still hit exist.com's DNS, which probably isn't terribly polite
I'd suggest since we're changing it, we should change it to doesntexist.ibm.com

Reply:
Ahhh of course! Will fix that and update. Thank you :)